include/klee/Expr/Expr.h

```diff
@@ -23,6 +20,8 @@
 #include <set>
 #include <vector>
 #include <map>
+#include "klee/Support/Bits.h"
+#include "Ref.h"
```

Comment:
For consistency that could be `#include "klee/Expr/Ref.h"`. I'm not sure though. What's our policy of library (module) local includes?